Demo/AppDelegate.swift

```diff
@@ -13,34 +13,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   var window: UIWindow?
 
-
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-    // Override point for customization after application launch.
+    window = UIWindow(frame: UIScreen.main.bounds)
+    let demoVC = DemoViewController()
+    window?.rootViewController = demoVC
+    window?.makeKeyAndVisible()
     return true
   }
 
-  func applicationWillResignActive(_ application: UIApplication) {
-    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-    // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
-  }
+  func applicationWillResignActive(_ application: UIApplication) {}
```

Comment:
I'd just get rid of these methods entirely...No point in keeping them unless we make use of them